Document/0x06b-Basic-Security-Testing.md

```diff
@@ -215,6 +215,14 @@ $ docker pull opensecurity/mobile-security-framework-mobsf
 $ docker run -it -p 8000:8000 opensecurity/mobile-security-framework-mobsf:latest
 ```
 
+Once you have MobSF up and running you can open it in your browser by navigating to <http://127.0.0.1:8000>. Simply select the IPA you want to analyze and MobSF will start its job. The bigger the app the longer it takes, but usually you should get some feedback within a few minutes.
+
+After MobSF is done with its analysis, you will receive a one-page overview of all the tests that were executed. While it may look daunting at first, the page is split up into multiple sections, each with their own purpose. Together, all the sections give a good first indication of the attack surface of the application. You can also execute additional actions, such as:
+
+- Download a class-dump, if the app was written in Objective-C; if it is written in Swift no classdump can be created.
```

Comment:
Have you tried running outside the docker container? You get better results for both swift and objc if you run it just with python. Try it :)

Reply:
I've added both options now and put a comment for that: "> By running it locally on a macOS host you'll benefit from a slightly better class-dump output."

The output is a bit cleaner in fact.